apm-agent-plugins/apm-slf4j-plugin/src/test/java/co/elastic/apm/agent/slf4j/Slf4JMdcActivationListenerTest.java

```diff
@@ -76,6 +79,33 @@ void testMdcIntegration() {
         transaction.end();
     }
 
+    @Test
+    void testDisabledWhenInactive() {
+        when(loggingConfiguration.isLogCorrelationEnabled()).thenReturn(true);
```

Comment:
this seems like two tests in one

Reply:
It's a positive testing proving that the following negative testing works.
You prefer I remove the positive test?

Reply:
Either remove it completely if it's already tested by other tests or make a separate test out of it.